README.md

```diff
@@ -53,6 +55,8 @@ Set your credentials and defaults in `config/smsapi.php`:
         // 'username' => env('SMSAPI_AUTH_USERNAME'),
         // 'password' => env('SMSAPI_AUTH_PASSWORD'), // Hashed by MD5
     ],
+    // TLD of smsapi. Can be PL or COM.
+    'service_domain' => 'PL',
```

Comment:
I think you might have forgotten to update this?

Reply:
Thanks. Updated it.